plugins/linux-v4l2/v4l2-input.c

```diff
@@ -1162,6 +1162,8 @@ static void *v4l2_create(obs_data_t *settings, obs_source_t *source)
 
 	signal_handler_connect(sh, "device_added", &device_added, data);
 	signal_handler_connect(sh, "device_removed", &device_removed, data);
+#else
```

Comment:
This seems like it should be a build time message. Like `message(STATUS "OBS: DISABLED UDEV support in linux-v4l2")`

Reply:
Should there be any message logged at runtime at all for this? Or would it make no sense to do so?

Reply:
I lean towards fewer things being dumped into the log, its not unreasonable to log it though. I think it would probably be best to just make udev a hard requirement.

Reply:
Would it be acceptable to land this as-is and have you or someone else make further adjustments later?

Reply:
Sure

Reply:
Okay. Give an approval review if this is good to land other than needing squashed. If this isn't rebased and squashed by Monday, I'll do that and then merge it. Feel free to put together the PR for further changes.